src/chrome/content/rules/LastPass.xml

```diff
@@ -1,11 +1,8 @@
-<ruleset name="LastPass.com" default_off="Breaks addon ( https://www.eff.org/r.3Dq )">
+<ruleset name="LastPass.com">
   <target host="lastpass.com" />
   <target host="*.lastpass.com" />
 
   <securecookie host="^(?:.+\.)?lastpass\.com$" name=".*"/>
 
-  <rule from="^http://(?:www\.)?lastpass\.com/" to="https://lastpass.com/"/>
-
-	<rule from="^http://forums\.lastpass\.com/"
-		to="https://forums.lastpass.com/" />
+  <rule from="^http:" to="https:" />
```

Comment:
Which other subdomains have you tested? You should test at least three, preferably more (you can often find subdomains using Wolfram Alpha). You should also add those subdomains as `<test url=... />`.